bcl2fastq/index_summary.py

```diff
@@ -254,34 +173,29 @@ def update_unknown_index_bar(run_json):
     }
 
 
-@app.callback(
-    [dep.Output('known_unknown_pie', 'figure'),
-     dep.Output('known_fraction', 'value')],
-    [dep.Input('run_select', 'value'),
-     dep.Input('sample_run_hidden', 'children'),
-     dep.Input('pruned_unknown_hidden', 'children')]
-)
-def update_pie_chart(run_alias, known_json, unknown_json):
-    """ When inputs are changed function is rerun to get updated pie chart
-            Parameters:
-                run_alias(str): alias of run
-                Known_json: json format of known indices data
-                unknown_json: json format of unknown indices data
-            Returns:
-                updated pie chart "known_unknown_pie" with known and unknown indices ratio over total cluster
-                updates value of known_fraction
-    """
-    known = pandas.read_json(known_json, orient='split')
-    pruned = pandas.read_json(unknown_json, orient='split')
+def update_pie_chart(run_alias):
+    """ Function to update pie chart and known fraction according to user selected run
+             Parameters:
+                 run_alias(str): name of run
+             Returns:
+                 updated pie chart "known_unknown_pie" with known and unknown indices ratio over total cluster
+                 updates value of known_fraction
+     """
+    known = index[index['Run'] == run_alias]
+    known = known[known['ReadNumber'] == 1]
+    known = known[~known['SampleID'].isna()]
+    known = known.drop_duplicates(['SampleID', 'LaneNumber'])
```

Comment:
`known` makes me think that this variable should hold something about the known indices, but lines 107-110 above have the exact same functionality, but calls the variable `run`. Can this be standardized to prevent confusion?